Tests/RxCocoaTests/NSTextView+RxTests.swift

```diff
@@ -49,7 +49,9 @@ extension NSTextViewTests {
                 name: NSText.didChangeNotification,
                 object: textView,
                 userInfo: nil)
-            textView.delegate?.textDidChange?(notification)
+            if let delegate = textView.delegate {
+                delegate.textDidChange?(notification)
```

Comment:
Do you really need to be explicit about this? Optional chaining should work 🤔

Reply:
Oh I pushed a wrong one... lemme fix this.